sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -1632,24 +1630,18 @@ def test_enet_cv_sample_weight_sparse(estimator):
         reg.fit(X, y, sample_weight=sw)
 
 
-@pytest.mark.parametrize("backend", ["loky", "threading"])
-@pytest.mark.parametrize(
-    "estimator", [ElasticNetCV, MultiTaskElasticNetCV, LassoCV, MultiTaskLassoCV]
-)
-def test_linear_models_cv_fit_for_all_backends(backend, estimator):
-    # LinearModelsCV.fit performs inplace operations on input data which is
-    # memmapped when using loky backend, causing an error due to unexpected
+@pytest.mark.parametrize("estimator", [ElasticNetCV, LassoCV])
+def test_linear_models_cv_fit_with_joblib(estimator):
+    # LinearModelsCV.fit performs inplace operations on fancy-indexed memmapped
+    # data when using the loky backend, causing an error due to unexpected
     # behavior of fancy indexing of read-only memmaps (cf. numpy#14132).
 
-    if parse_version(joblib.__version__) < parse_version("0.12") and backend == "loky":
-        pytest.skip("loky backend does not exist in joblib <0.12")
-
     # Create a problem sufficiently large to cause memmapping (1MB).
-    n_targets = 1 + (estimator in (MultiTaskElasticNetCV, MultiTaskLassoCV))
-    X, y = make_regression(20000, 10, n_targets=n_targets)
-
-    with joblib.parallel_backend(backend=backend):
-        estimator(n_jobs=2, cv=3).fit(X, y)
+    # Unfortunately the scikit-learn and joblib APIs do not make it possible to
+    # change the max_nbyte of the inner Parallel call.
```

Comment:
Yes that would be nice but unfortunately this conflicts with the current joblib backend API design and dealing with backward compat is...

Reply:
If it is hard for joblib API wise, what do you think about adding a `parallel_kwargs` parameter to estimators that creates a `Parallel` object?

(I know this is a little counter to how we have been removing `pre_dispatch` from the estimator's `__init__`.)

Reply:
We should probably find a way to change the `parallel_backend` to detect `Parallel`  kwargs and treat time specifically instead of passing them as constructor arguments for the backend.